source/module_base/module_container/ATen/ops/linalg_op.h

```diff
@@ -137,7 +137,7 @@ struct inflate_op {
      */
     void operator()(
         const Tensor& input,
-        const std::vector<int64_t>& stride,
+        const std::vector<int64_t>& inflate,
```

Comment:
what's the reason to change the variable's name?

Reply:
This is because the corresponding variable name in the implementation of the function in the `.cpp` file is `inflate`. So, it is to keep the declaration and implementation of the function consistent. 

LGTM.